ui/app/AppLayouts/Profile/views/MyProfileView.qml

```diff
@@ -253,5 +269,11 @@ SettingsContentBase {
                 onClosed: destroy()
             }
         }
+
+        Component {
+            id: profileShowcaseInfoPopup
+
+            ProfileShowcaseInfoPopup {}
```

Comment:
pls add:
```qml
destroyOnClose: true
```
in order not to leak the memory

Reply:
Okk!!